src/textalloc/__init__.py

```diff
@@ -602,3 +604,20 @@ def get_scatter_bbs(sc, ax) -> np.ndarray:
             bboxes.append(result.bounds)
 
     return np.array(bboxes)
+
+
+# Based on https://stackoverflow.com/questions/22667224/get-text-bounding-box-independent-of-backend/
+def _get_renderer(fig: mpl.figure.Figure) -> mpl.backend_bases.RendererBase:
```

Comment:
mpl.figure.Figure fails for me with "AttributeError: module 'matplotlib' has no attribute 'figure'", either remove the typing or import Figure using:
from matplotlib.figure import Figure

Reply:
Fixed in https://github.com/ckjellson/textalloc/pull/40/commits/5c88636b5e4bb2d10153c2ac476d49dfe44e1330